src/dapps/eth-blocks/components/BlockSearch.vue

```diff
@@ -63,7 +63,7 @@ export default {
     maxBlock() {
       const max = toBN(this.blockNumber).sub(toBN(50));
       const ZERO = toBN(0);
-      return max.gt(ZERO) ? max.toNumber() : '';
+      return max.gt(ZERO) ? max : '';
```

Comment:
removing .toNumber() will break the ui as `maxBlock` is being rendered on line 8